source/_components/egardia.markdown

```diff
@@ -59,9 +65,45 @@ You can change this, however, using the following procedure. This is a more adva
         triggered: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
         ignore: XXXXXXXXXXXXXXXX
     ```
+{% configuration %}
```

Comment:
Only one of these sections should be written and it should contain all the options.

Reply:
ok